Configuration/PyReleaseValidation/python/relval_standard.py

```diff
@@ -457,8 +457,26 @@
 workflows[139.005] = ['',['AlCaPhiSym2021','RECOALCAECALPHISYMDR3','ALCAECALPHISYM']]
 
 ### run3 (2022) ###
-workflows[140.001] = ['',['RunMinimumBias2022B','HLTRUN3','RECONANORUN3','HARVESTRUN3']]
-workflows[140.002] = ['',['RunZeroBias2022B','HLTRUN3','RECONANORUN3','HARVESTRUN3']]
+workflows[140.001] = ['',['RunZeroBias2022B','HLTRUN3','RECONANORUN3','HARVESTRUN3']]
```

Comment:
Do we need to switch the workflow of ZeroBias and MinBias here? This will make failure in comparison of the workflow test.

Reply:
No, actually this is totally accidental and realized after creating PR.

Reply:
OK, Please fix. Thanks.

Reply:
@srimanob I made the changes. Please see if these are correctly propagated to the PR.